ciao-controller/controller_test.go

```diff
@@ -732,22 +731,21 @@ func testStartWorkloadLaunchCNCI(t *testing.T, num int) (*testutil.SsntpTestClie
 	newTenant := uuid.Generate().String() // random ~= new tenant and thus triggers start of a CNCI
 
 	// trigger the START command flow, and await results
-	// NOTE: "instances" is shared with the go subroutine and we must
-	// insure consistency between parent and child processes
-	var instances []*types.Instance
+	instanceCh := make(chan []*types.Instance, 1)
```

Comment:
I've inadvertantly created a buffered channel here.

instanceCh := make(chan []*types.Instance, 1)
is not
instanceCh := make(chan []*types.Instance, 0)
which is
instanceCh := make(chan []*types.Instance)